migration/v0.46/apply/10-gatekeeper-crds.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "installing gatekeeper crds in service cluster"
+      helmfile_do sc -l name=gatekeeper-templates apply
```

Comment:
```suggestion
      helmfile_do sc -l name=gatekeeper-templates apply
      helmfile_upgrade sc app=gatekeeper
```

Could be good to do the Gatekeeper upgrade in this script as well, and [excluding it from the general apply](https://github.com/elastisys/compliantkubernetes-apps/blob/alb/upgrade-gatekeeper/migration/v0.46/apply/80-apply.sh#L11), thoughts?

Reply:
Sounds good, PTAL ☺️